### PR TITLE
Added M1 compatibility

### LINKED ISSUE #220: M1 support?
Any plans for an M1 build?

Or a universal binary, as far as I can tell it's only intel so far. (though does seem to work ok via Rosetta)

### PR DESCRIPTION
**Pull request by erseco**
Added apple silicon universal support as mentioned here https://github.com/electron-userland/electron-builder/issues/5689 to fix #220

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,11 +96,14 @@
...
       "gatekeeperAssess": false,
       "entitlements": "./build/entitlements.mac.plist",
-      "entitlementsInherit": "./build/entitlements.mac.plist",
-      "target": [
-        "dmg",
-        "zip"
-      ]
+      "entitlementsInherit": "./build/entitlements.mac.plist",      
+      "target": {
+        "target": [
```
Comment (ganeshrvel) [inline package.json:101]: this wouldnt work.

`target` nested inside `target`?

**Comment by jiahao-c** [comment]
Looking forward to seeing this getting merged or fixed:)

**Comment by ganeshrvel** [comment]
Check this comment for the [M1 version](https://github.com/ganeshrvel/openmtp/issues/220#issuecomment-1050236345)

**Comment by ganeshrvel** [comment]
Closing this PR as there another PR tracking the M1 release

**Pull request closed without merging**